src/tree/activityLog/ActivityItem.ts

```diff
@@ -65,7 +65,7 @@ export class ActivityItem implements TreeElementBase, Disposable {
         }
     }
 
-    public initialCollapsibleState: TreeItemCollapsibleState = TreeItemCollapsibleState.None;
+    public initialCollapsibleState: TreeItemCollapsibleState = TreeItemCollapsibleState.Expanded;
```

Comment:
Can we ship this without it being initially set to expanded?  There are a couple scenarios in container apps that look kind of weird without the progress implementations.  I plan to address this once we migrate that extension to V2, but for initial pass maybe we only show the children at the end?